src/Audacia.Core/EnumMember.cs

```diff
@@ -285,6 +288,39 @@ public static class EnumMember
             throw new ArgumentException($"Requested value '{value}' was not found.");
         }
 
+        /// <summary>
+        /// Attempts to parse the enum as a string and return the relevant member.
+        /// </summary>
+        /// <param name="type">Type of Enum.</param>
+        /// <param name="value">Value to convert.</param>
+        /// <param name="enumValue">Return value of the enum if parsed.</param>
+        /// <returns>The parsed enum.</returns>
+        /// <exception cref="OverflowException"> Throws when the enum member is out of range.</exception>
+        public static bool TryParseEnumAsString(Type type, string value, out object? enumValue)
+        {
+            bool enumDefined;
+
+            try
+            {
+                // Match by number value or field name
+                enumValue = Enum.Parse(type, value, ignoreCase: true);
```

Comment:
I think you can use `Enum.TryParse` here and eliminate the `try-catch`. Would be a bit more efficient.

Reply:
both Enum.IsDefined and Enum.Parse can throw argument exceptions so I believe this was a catch all for both